R/median_normalisation.R

```diff
@@ -1,29 +1,36 @@
 #' Median normalisation
 #'
-#' Median normalise intensity values for all samples of mass spectrometry data. The intensity values are ideally log2 transformed. The output is a new column called normalised_intensity_log2.
+#' Normalises the intensities in the way this is implemented in Spectronaut. The normalised intensity is the original intensity minus the run median plus the global median.
 #'
-#' @param data A data frame containing at least sample names and intensity values.
-#' @param file_column_name The name of the column containing the sample names.
-#' @param intensity_column_name The name of the column containing the intensity values.
-#' @param na.rm A logical indicating wther missing values should be removed.
+#' @param data A data frame containing at least sample names, grouping variables and intensity values.
+#' @param sample The name of the column containing the sample names.
+#' @param grouping The name of the column containing the grouping variable - this can be peptides, precursors or proteins.
+#' @param log2intensity The name of the column containing the log2 transformed intensity values to be normalised.
```

Comment:
maybe call the variable intensity_log2 or log2_intensity. I think this is what we called it before somewhere.